src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkDemandIndex/__tests__/MyCollectionArtworkDemandIndex.jest.tsx

```diff
@@ -0,0 +1,106 @@
+import { screen } from "@testing-library/react"
+import { MockBoot } from "DevTools"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "relay-runtime"
+import { MyCollectionArtworkDemandIndexTestQuery } from "__generated__/MyCollectionArtworkDemandIndexTestQuery.graphql"
+import { MyCollectionArtworkDemandIndexFragmentContainer } from ".."
+
+jest.unmock("react-relay")
+
+describe("MyCollectionArtworkDemandIndex", () => {
```

Comment:
**issue:** since the demand rank logic is now in metaphysics, we don't need to test for cases where it's less, equal or bigger than 9.

Reply:
yeah forgot about this test 😅 I'll update it now!